### PR TITLE
Create types packages for "Stats" and "Visualizer" services

### PR DESCRIPTION
**Pull request by tom2drum**
As the integration for BENS types package now works smoothly, I've added the same code for the rest of the services used by the frontend app ("Stats", "Visualizer").

### CODE REVIEW

File: blockscout-ens/types/.gitignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
-/node-modules
+/node_modules
```
Comment (rimrakhimov) [inline blockscout-ens/types/.gitignore:1]: Can we add node_modules to the root `.gitignore` instead to be ignored recursively?

**Review by rimrakhimov — APPROVED** [review]

File: blockscout-ens/types/.gitignore (hunk 1 above)
Comment (tom2drum) [reply blockscout-ens/types/.gitignore:1]: Currently there are 17 occurrences of "node_modules" in the project's `.gitignore` files. Should I remove them all in favor of the root file?

image

Comment (rimrakhimov) [reply blockscout-ens/types/.gitignore:1]: I think so

Comment (tom2drum) [reply blockscout-ens/types/.gitignore]: Done ✅

**Comment by bragov4ik** [comment]
@rimrakhimov sourcify test is acting up again 😢

**Pull request merged**